test/test_bash_functions.py

```diff
@@ -18,7 +19,11 @@ def test_IPv6_not_True_removes_ipv6(Docker, args, expected_ipv6, expected_stdout
 
     function = Docker.run('. /bash_functions.sh ; setup_ipv4_ipv6')
     assert "Using {}".format(expected_stdout) in function.stdout
-    config = Docker.run('cat {}'.format(WEB_CONFIG)).stdout
+    if expected_stdout == 'IPv4':
+        assert 'IPv6' not in function.stdout
+    # weird slow write/sync problem; no sleep == old state of file, sleep == updated/setup state of file
+    time.sleep(1)
```

Comment:
`setup_ipv4_ipv6` does what it is supposed to do according to every manual test I perform but automation/docker seems to read the file from the old state unless you wait a small amount of time at which point